db/blob/blob_source.h

```diff
@@ -123,6 +123,27 @@ class BlobSource {
     return base_cache_key.WithOffset(offset);
   }
 
+  // Callbacks for secondary blob cache
+  static size_t SizeCallback(void* obj) {
+    assert(obj != nullptr);
+    return static_cast<std::string*>(obj)->size();
```

Comment:
Minor but you could also cast to `const std::string*` here

Reply:
Done